doc/mgmt/SONiC_OC_MGMT_Interface_HLD.md

```diff
@@ -182,7 +182,6 @@ module: openconfig-interfaces
 ```
 ### 3.6.2 CLI
 #### 3.6.2.1 Configuration Commands
-1. interface Management <Interface Id>
 
   sonic(config)# interface ?
   Ethernet    Interface commands
```

Comment:
ethernet/management to be lowercase

Reply:
Currently cli accepts both upper and lower case letters. If we need to change to only lower case it will be user interface change as this CLI syntax is committed as part of arlo. Cant be changed at this point.